lib/streamlit/elements/multiselect.py

```diff
@@ -264,6 +291,13 @@ def _multiselect(
             serializer=serde.serialize,
             ctx=ctx,
         )
+        if max_selections and _get_default_count(widget_state.value) > max_selections:
```

Comment:
nit: could assign `_get_default_count(widget_state.value)` to a variable to prevent the duplicated function calls